build.gradle

```diff
@@ -9,6 +9,7 @@ buildscript {
     dependencies {
         classpath 'ru.vyarus:gradle-animalsniffer-plugin:1.5.0'
         classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.16'
+        classpath "net.ltgt.gradle:gradle-apt-plugin:0.18"
```

Comment:
Based on this comment this should not be needed https://github.com/tbroyer/gradle-apt-plugin#gradle-apt-plugin

Reply:
I see, didn't realize this project is using Gradle 5.3.